frontend/src/container/GridGraphLayout/Graph/index.tsx

```diff
@@ -256,7 +257,29 @@ function GridCardGraph({
 					onView={handleOnView}
 					onDelete={handleOnDelete}
 				/>
-				<Spinner height="20vh" tip="Loading..." />
+				{prevChartDataSetRef.current === undefined && (
+					<Spinner
+						height="12vh"
+						tip="Loading..."
```

Comment:
we can reduce the size of the spinner also

Reply:
Already reduced from 20 to 12vh.
Do you want me to still reduce?

Reply:
that's the height of rendering os spinner component i am referring size prop from spinner

Reply:
That is default loader on render of page. once you change time interval then we are showing only loader without tip prop.
So automatically showing smaller than current. Can you please recheck i updated my logic.